### DISCUSSION TITLE
Multiple Instances of ChatOps on Single Stackstorm Instance

### DISCUSSION THREAD

**Discussion by rdancel001** [StackStorm/st2#5683] [ChatOps] [answered]
Has this been done before? I have a single stackstorm instance and would like to have multiple chat bots in different Slack Communities with the understanding that it'll also be tied to multiple MS Teams .. Teams/Groups. That way we can have the bot available for the different teams we support, and document a subset of functions for each team to be able to execute.

**Comment by nzlosh** [comment, marked as answer]
hubot-stackstorm doesn't support this feature.

Today, it's possible to run multiple instances of err-stackstorm against a single stackstorm instance.  It uses route keys to achieve this.  https://err-stackstorm.readthedocs.io/en/latest/configuration.html#route-key  That said, err-stackstorm doesn't support MSTeams because I don't have access to it for development.  Slack and many other chat services are supported however.

At some time in the future an opsdroid integration will be developed and I'm sure it'll include support for this feature.

**Comment by rdancel001** [comment]
It was going so good until you mentioned no support for MSTEams hahah. Our end goal is MS Teams since its our enterprise chat app. But, nonetheless this is good to know. Sounds like I'll have to put this in a backlog ticket then and revisit it again.
